.gitignore

```diff
@@ -38,5 +38,5 @@ celeryev.pid
 media/js/libs/ckeditor/source/ckbuilder
 .tox
 .env
-*_devmo.sql*
+*devmo*.sql*
```

Comment:
There does exist the ability to have initial SQL files that Django runs to do database prep like stuff.